atomics/T1046/T1046.yaml

```diff
@@ -47,6 +47,6 @@ atomic_tests:
   executor:
     name: sh
     command: |
-      nmap -sS #{network_range} -p #{port}
+      if [ -x "$(command -v nmap)" ]; then nmap -sS #{network_range} -p #{port}; else echo "nmap is missing from the machine. skipping..."; fi;
```

Comment:
Could we add the check for binary existing as actual prereqs as you do in some of the later files in this commit?

Reply:
point taken